server/src/main/java/org/elasticsearch/index/fielddata/ScriptDocValues.java

```diff
@@ -48,6 +53,25 @@
  */
 public abstract class ScriptDocValues<T> extends AbstractList<T> {
 
+    private static final DeprecationLogger deprecationLogger = new DeprecationLogger(ESLoggerFactory.getLogger(ScriptDocValues.class));
```

Comment:
Could you use `LogManager.getLogger` instead of `ESLoggerFactory`. I'm making that not a thing soon.

Reply:
Will do! I was back and forth on this.

Reply:
Done.